yt/yt/client/table_client/wire_protocol.h

```diff
@@ -280,13 +280,24 @@ struct IWireProtocolReader
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct TWireProtocolOptions {
+    ui32 MaxStringValueLength = NYT::NTableClient::MaxStringValueLength;
+    ui32 MaxAnyValueLength = NYT::NTableClient::MaxAnyValueLength;
+    ui32 MaxCompositeValueLength = NYT::NTableClient::MaxCompositeValueLength;
+    ui32 MaxTimestampCountPerRow = NYT::NTableClient::MaxTimestampCountPerRow;
+    size_t MaxVersionedRowDataWeight = NYT::NTableClient::MaxServerVersionedRowDataWeight;
```

Comment:
Why `size_t`, the original type of this constant is `i64`, let use it

Reply:
Because we use it to compare to a [size_t variable](https://github.com/ytsaurus/ytsaurus/pull/943/files#diff-47ef756916285c1a2b1d1319972480fef75bdd221e592430c9740f3ec35644fbR1010). [GetDataWeight](https://github.com/ytsaurus/ytsaurus/blob/6a32092f9311afdcc0ee7bb46c292cda97775411/yt/yt/client/table_client/versioned_row.cpp#L24) returns `size_t`

Same for other variables

Originally I did i64 everywhere, but it will require casting one of them for comparison. either `size_t` to `i64`, or the other way

Reply:
Changed all `ui32` to `i64`

Kept `size_t`. It would require casting either `size_t` or `i64` on [this line](https://github.com/ytsaurus/ytsaurus/pull/943/files#diff-47ef756916285c1a2b1d1319972480fef75bdd221e592430c9740f3ec35644fbR1011)

Reply:
I suppose to use i64 for all these fields